templates/_linux.yaml

```diff
@@ -66,7 +70,16 @@ objects:
             - disk:
                 bus: {{ diskbus | default("virtio") }}
               name: rootdisk
+            - disk:
```

Comment:
Wasn't this disk added by default by kubevirt core components? But the truth is I never noticed we have volume and no disk...

Reply:
The disk does not get add... it doesn't complain about it not being there. The machine just hangs for a long time at boot and if this disk is added to the template the machine boots as expected and cloud-init is processed.